src/components/Filter/components/FilterSectionHeader/filterSectionHeaderStyles.ts

```diff
@@ -2,7 +2,7 @@ import styled from '@emotion/styled';
 import { ReactComponent as CloseLabel } from '../../assets/closeLabel.svg';
 
 export const FilterValueChip = styled.div(({ theme }) => ({
```

Comment:
Some styling change should be made to avoid the chip changing height when there is an "x" vs when there isn't an "x". Honestly, I'm not sure added height is needed when there is an "x" present (i.e., keep the smaller height if at all possible).

![image](https://user-images.githubusercontent.com/80778550/214610936-9d8c92c5-b427-4522-b67c-32d1a25f0bc9.png)
![image](https://user-images.githubusercontent.com/80778550/214610835-021aeba4-c60d-42a5-8d84-b320115b275c.png)